optuna/integration/wandb.py

```diff
@@ -0,0 +1,142 @@
+from typing import Optional
+from typing import Union
+
+import optuna
+from optuna._imports import try_import
+
+
+with try_import() as _imports:
+    import wandb
+
+
+class WeightsAndBiasesCallback(object):
```

Comment:
Should I decorate it with `@experimental("2.9.0")`?

Reply:
Yes! Thanks!